src/plugins/interactive_setup/public/theme/kibana_theme_provider.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiThemeProvider } from '@elastic/eui';
+import type { FC } from 'react';
+import React, { useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import type { Observable } from 'rxjs';
+
+import type { CoreTheme } from '../../../../core/public';
+import { getColorMode } from './utils';
+
+interface KibanaThemeProviderProps {
+  theme$: Observable<CoreTheme>;
+}
+
+const defaultTheme: CoreTheme = {
+  darkMode: false,
+};
+
+/**
+ * Copied from the `kibana_react` plugin, remove once https://github.com/elastic/kibana/issues/119204 is implemented.
```

Comment:
__question:__ Should we mirror this comment (and the one in `utils` as well) in `kibana_react` with `IMPORTANT:` and ask to keep these two in sync in case original `KibanaThemeProvider` is updated?

Reply:
Good call, I addressed this and your other comment in 9de02702928f381bae32b6f484545e61c3cb4b00.